tests/operator/operator.go

```diff
@@ -2527,12 +2527,9 @@ spec:
 			monv1 := virtClient.PrometheusClient().MonitoringV1()
 			prometheusRule, err := monv1.PrometheusRules(flags.KubeVirtInstallNamespace).Get(context.Background(), components.KUBEVIRT_PROMETHEUS_RULE_NAME, metav1.GetOptions{})
 			Expect(err).ShouldNot(HaveOccurred())
-			hasWorkloadUpdates := false
-			if len(originalKv.Spec.WorkloadUpdateStrategy.WorkloadUpdateMethods) > 0 {
-				hasWorkloadUpdates = true
-			}
```

Comment:
Please keep these lines, they were removed in main as part of https://github.com/kubevirt/kubevirt/commit/61a98f1267b0a8cfb8d879fa296c4c75556dfb4b but this was not backported.

Reply:
But `hasWorkloadUpdates` isn't in use after removing `NewPrometheusRuleSpec()`.

Reply:
You are right, never mind.